src/components/views/elements/AppTile.js

```diff
@@ -56,6 +59,7 @@ export default React.createClass({
     },
 
     componentWillMount: function() {
+        this._onEditClick = this._onEditClick.bind(this);
```

Comment:
Yeah, I thought so too, but was wrangling issues where `this._scalarClient` was not defined and I was clutching at straws to see if it was a bind problem (it was due to something else). I should've removed this before committing, thanks.